lib/netflix2.class.js

```diff
@@ -242,15 +281,24 @@ class Netflix {
     await this.__setRating(false, titleId, rating)
   }
 
+  /**
+   *
+   * @param {number} titleId
+   * @param {string} rating
```

Comment:
This is incorrect, ratings are also numbers. More precicely, the value is 1 for "thumbs down" and 2 for "thumbs up".

Reply:
@LBBO true, the whole file is dirty now because of semicolons. Will get right on it